src/webview/pipeline/app/utils/types.ts

```diff
@@ -15,7 +15,7 @@ export interface NameType {
 
 export interface ItemValue {
   key?: string;
-  value?: string;
+  path?: string;
```

Comment:
In this case you need to change type name `ItemValue` -> `ItemPath`, or keep `value`

In other hand we already have absolutely the same type `ItemValue`

Reply:
in workspace items contain `key` and `path` so I change the value to `path`